Add support for recursive compiling

mikemartin opened the pull request:
Added recursive compiling support as mentioned in https://github.com/matejsvajger/laravel-mix-pug/issues/7

Recursive compiling fix from https://github.com/rivergod/laravel-mix-pug/commit/c5658a2420a7a05f10dfb586d4ad9c61552363e2.

mikemartin committed:
recursive compile

mikemartin committed:
update readme

mikemartin committed:
update readme

mikemartin committed:
update package

dsktschy commented:
@mikemartin
Both #14 and #15 solve the same problem.
I think #15 is more simple and better because it does not increase the new dependencies.
I hope pull requests will be integrated into # 15 only.
Thanks!

RikutoYamaguchi committed:
Add glob option at glob.sync

RikutoYamaguchi committed:
Bugfix

sebszocinski committed:
Update package.json

Update package peer dependency to latest version of Laravel Mix

mikemartin committed:
Update package.json

RikutoYamaguchi committed:
Bugfix issue-3

mikemartin committed:
Merge pull request #5 from RikutoYamaguchi/base_master

Bugfix issue-3

matthewjumpsoffbuildings committed:
Update PugSeeder.js

fix missing extname require

mikemartin committed:
Merge pull request #7 from matthewjumpsoffbuildings/master

mikemartin committed:
Merge branch 'matejsvajger:master' into master

mikemartin committed:
Bump npm version

evmason committed:
Fix "Assert.dependencies is not a function"

starting from version 6.0.38 laravel-mix/Assert.js remove "dependencies" function

evmason committed:
Merge pull request #1 from evmason/evmason-patch-1

Fix "Assert.dependencies is not a function"

mikemartin committed:
Merge pull request #10 from evmason/master
